06-instructions.md

```diff
@@ -60,7 +59,7 @@ For instance, if the client is Bob who initially owns BTC in a swap, and the can
 
 ## Instructions: Low level
 
-`instructions` must convey all required data a daemon needs to fulfill its mission, not only the keys or the transactions' signatures but also the user's commands for continuing the swap or cancelling it.
+`instructions` must convey all required data a daemon or a client needs to fulfill its mission, not only the keys or the transactions' signatures but also the user's commands for continuing the swap or cancelling it.
```

Comment:
why clients too? is this for inter-client communication (for the same swap party)?

Reply:
nvm - missed the overall change in naming scheme

Reply:
the name instruction is a bit misleading there, and breaks the previous concept u probably have in mind

now instruction is all messages exchanged by client and daemon -- in need for a better name